deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java

```diff
@@ -1563,12 +1563,20 @@ protected void truncatedBPTTGradient() {
 
     /**
      *
-     * @return
+     * @return listeners
      */
     public Collection<IterationListener> getListeners() {
         return listeners;
     }
 
+    /**
+    *
+    * @return trainingListeners
+    */
+   public Collection<trainingListeners> getTrainingListeners() {
+       return trainingListeners;
```

Comment:
This doesn't look right. Shouldn't this be camel case? We don't have all lower case classes.

Reply:
I don't see. First word (get) should have small letter. Next word (Training) should have capital letter to separate sense. Last word (Listeners) should have capital letter to separate sense and it's first capital letter correspond with collection name. For me it is standard name.
In MultiLayerNetwork I see a lot of similar.

Reply:
The method name is fine, but ```Collection<trainingListeners>``` won't compile. It should be ```Collection<TrainingListener>``` instead.